pkg/i2gw/ingress2gateway.go

```diff
@@ -42,7 +42,15 @@ func Run() {
 		os.Exit(1)
 	}
 
-	outputResult(ingresses2GatewaysAndHttpRoutes(ingressList.Items))
+	httpRoutes, gateways, errors := ingresses2GatewaysAndHttpRoutes(ingressList.Items)
+	if len(errors) > 0 {
+		fmt.Printf("# Encountered %d errors\n", len(errors))
+		for _, err := range errors {
+			fmt.Printf("# %s", err)
+		}
+	}
```

Comment:
Maybe errors should also be output in JSON/YAML form?

Reply:
Interesting thought. Which fields would you suggest adding to the JSON/YAML, in addition to an "errors" array? If you have an example, it would be great to see one.

Reply:
No strong preference what it looks like, just if a user asks for json or yaml output it seems ideal to provide that format, even if the output is just a list of errors. Though I can be convinced against this, I can't remember what kubectl does here.

Reply:
The default output format right now is YAML, which means that whether or not I specify an output format, I will never get a plain error message (always YAML/JSON). Personally, I can't think of any CLI tool that displays error messages in this way, especially in the Kubernetes ecosystem, but I may be wrong.

Overall, I'm not so sure about it. However, if you are up for it, then there's no problem.

Reply:
yeah that's fair, I guess most CLI tools don't output error messages in yaml or JSON